R/pileup.R

```diff
@@ -56,8 +56,10 @@
 #' @importFrom GenomicRanges GRanges
 #' @importFrom IRanges IRanges
 #' @importFrom GenomeInfoDb seqlevels seqinfo seqlengths
-#' @importFrom BiocParallel SerialParam bpstop bplapply
-#' @rdname get_pileup
+#' @importFrom BiocParallel SerialParam bpstop bplapply ipcid ipclock ipcunlock
+#'
+#' @family pileup
+#'
 #' @export
 get_pileup <- function(bamfiles,
```

Comment:
We should consider renaming this function to something more descriptive like `pileup_sites`. Not necessary in this PR, but something to consider. We however can't use `pileup` as that would conflict with rsamtools.

Reply:
I'm fine with `pileup_sites`. Are the outputs similar between `pileup_sites()` and `pileup_cells()`?

Reply:
the outputs will be similar classes once I've merged #70. pileup_cells returns a SingleCellExperiment and pileup_sites will return a RangedSummarizedExperiment.

Reply:
if you want to merge this in, i'll incorporate the changes, and rename get_pileup in #70.